src/mds/CDir.cc

```diff
@@ -306,9 +313,10 @@ CDentry *CDir::lookup(std::string_view name, snapid_t snap)
 }
 
 CDentry *CDir::lookup_exact_snap(std::string_view name, snapid_t last) {
+  dout(20) << "lookup_exact_snap (" << last << ", '" << name << "')" << dendl;
```

Comment:
Use `__func__` instead of writing out the name.

Reply:
All changes is updated